MouseMovementTool.cs

```diff
@@ -13,16 +13,9 @@
 namespace UnityEditor.Experimental.EditorVR.Tools
 {
     sealed class MouseMovementTool : MonoBehaviour, ITool, ILocomotor, IRayVisibilitySettings,
-        /*ICustomActionMap,*/ ILinkedObject, IUsesViewerScale,
+        /*ICustomActionMap, ILinkedObject,*/ IUsesViewerScale,
         IUsesDeviceType, IGetVRPlayerObjects, IBlockUIInteraction, IRequestFeedback
     {
-        public List<ILinkedObject> linkedObjects
-        {
-            set
-            {
-                throw new NotImplementedException();
-            }
-        }
 
         public Transform cameraRig
```

Comment:
Where is the cameraRig info coming from?  Is an interface requiring it/filling it in?
I would expect something more like
public Transform cameraRig { private get; set; }

The getter right now will return itself which will make a stack overflow, and there is no way to put data into it.